portfolio/src/main/webapp/index.html

```diff
@@ -35,7 +35,7 @@ <h2>Shuli's Portfolio</h2>
     </div> 
     
     <div id="guest-book">
-      <form action="/guestbook?maxcomments=5" method="POST">
+      <form action="/guestbook?maxcomments=5&lang=en" method="POST">
```

Comment:
In your other files maxcomments has a dash in it which is missing here

Reply:
Nice catch!